.github/workflows/all.yml

```diff
@@ -0,0 +1,299 @@
+name: Deps, Lint, Test, Build, Publish
+on:
+  # Enable manually triggering this workflow via the API or web UI
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+    tags:
+      - v*
+  pull_request:
+
+env:
+  APP_NAME: "k6"
+  DOCKER_IMAGE_ID: ${{ github.repository }}
+
+jobs:
+  deps:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.14.x
```

Comment:
I am wondering if we can define the two versions ... in a variable thing above and just use the variable here so we have 
`current_version`
`previous_version`
and maybe `release_version`

Reply:
Yeah, that would be good, but it doesn't seem to work with the `env` context :( See https://github.com/imiric/k6/actions/runs/280602334 . Apparently this only works for each step's `env`, and not with `matrix` or anything else, so it's quite limited. See [this discussion](https://github.community/t/how-to-use-env-context/16975).

There doesn't seem to be something equivalent to [variables as in Azure Pipelines](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch), so I think we're stuck with repeating the version everywhere. :-/